<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>entry_window_title</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="ZoomScrollArea" name="scrollArea">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaCentralWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>301</width>
          <height>557</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget { background-color:gray; }</string>
        </property>
        <layout class="QVBoxLayout" name="scrollAreaLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>100</number>
         </property>
         <item>
          <widget class="WorkArea" name="workArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget { background-color:white; }</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="rightSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>4</number>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="pageListLayout">
         <item>
          <layout class="QHBoxLayout" name="pageListEditorLayout">
           <item>
            <widget class="QPushButton" name="pageListEdit">
             <property name="text">
              <string>mainWindow_pageListEdit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pageListEditDone">
             <property name="text">
              <string>mainWindow_pageListEditDone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pageListAdd">
             <property name="text">
              <string>mainWindow_pageListAdd</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="pageList">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="TranslationEditArea" name="translationEditArea" native="true">
        <property name="text" stdset="0">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>mainWindow_menuProject</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionShowTutorial"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>mainWindow_menuHelp</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionFeedback"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>mainWindow_actionNew</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>mainWindow_actionOpen</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>mainWindow_actionSave</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>mainWindow_actionSaveAs</string>
   </property>
  </action>
  <action name="actionShowTutorial">
   <property name="text">
    <string>mainWindow_actionShowTutorial</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>mainWindow_actionExit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>mainWindow_actionAbout</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>mainWindow_actionFeedback</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>mainWindow_actionAboutQt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomScrollArea</class>
   <extends>QScrollArea</extends>
   <header>include/ZoomScrollArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorkArea</class>
   <extends>QLabel</extends>
   <header>include/WorkArea.h</header>
  </customwidget>
  <customwidget>
   <class>TranslationEditArea</class>
   <extends>QWidget</extends>
   <header>include/TranslationEditArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
